### PR TITLE
Increase time for MultiDcSplitBrain and increase cross DC gossip prob

### PR DESCRIPTION
**Pull request by chbatey**
The test has been failing infrequently as when we get to the final
barrier (restarted-fifth-removed) the whole test withIn of 40s
has been reached so the last barrier times out right away.

Trying to remove the Thread.sleep and rely on a larger timeout for the
whole test as well as the default barrier timeout of 30s.

Also increase the cross DC probability to see if it speeds this test up, which could
be fixed in https://github.com/akka/akka/issues/23803. 

Local testing (5 times each) for this has the test runtime as:
* Default 0.2: 45-55s
* 0.8: ~40-50s
* 1.0: Slows it down so much that is constantly takes 60+s so always fails

Refs https://github.com/akka/akka/issues/23775

### CODE REVIEW

File: akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala [hunk 1]
```diff
@@ -32,6 +28,7 @@ object MultiDcSplitBrainMultiJvmSpec extends MultiNodeConfig {
       akka.remote.netty.tcp.connection-timeout = 5 s # speedup in case of connection issue
       akka.remote.retry-gate-closed-for = 1 s
       akka.cluster.multi-data-center {
+        cross-data-center-gossip-probability = 0.8
```
Comment (patriknw) [inline akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala]: increasing too much will cause low probability for gossip within same dc, did you try with 0.5?

Comment (chbatey) [reply akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala]: I didn't, trying now

Comment (chbatey) [reply akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala]: 0.5 is roughly the same as 0.8, whole test takes ~45s

**Comment by johanandren** [comment]
I wonder if this is the reason for #23879 as well, and if so, why did these start failing constantly since two days ago, can't find that we merged anything that should cause it, did we do something with Jenkins on the 31st that made it slower/more busy?

**Comment by chbatey** [comment]
I've removed the change of the cross dc probability as that should be fixed in master now. 

So this is now just the removal of the Thread.sleep and increasing the timeout

**Review by ktoso — APPROVED** [review]
LGTM, failure was jenkins having some weird issue in sending the status change, and there was the javac failure too.

**Comment by raboof** [comment]
PLS BUILD

**Comment by raboof** [comment]
PLS BUILD

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**